go/serialization.go

```diff
@@ -147,6 +147,10 @@ func ReadTimestampedEntryInto(r io.Reader, t *TimestampedEntry) error {
 		if t.PrecertEntry.TBSCertificate, err = readVarBytes(r, PreCertificateLengthBytes); err != nil {
 			return err
 		}
+	case XJSONLogEntryType:
```

Comment:
I kind of missed this in the previous round, but, er, this has nothing to do with "implementing GetProofByHash", doesn't it? :wink: